packages/app/src/app/pages/Sandbox/index.tsx

```diff
@@ -22,152 +22,157 @@ interface Props {
   };
 }
 
-export const Sandbox: React.FC<Props> = ({ match }) => {
-  const { state, actions } = useOvermind();
-
-  useEffect(() => {
-    if (window.screen.availWidth < 800) {
-      if (!document.location.search.includes('from-embed')) {
-        const addedSign = document.location.search ? '&' : '?';
-        document.location.href =
-          document.location.href.replace('/s/', '/embed/') +
-          addedSign +
-          'codemirror=1';
-      } else {
-        actions.preferences.codeMirrorForced();
+export const Sandbox: React.FC<Props> = React.memo(
```

Comment:
This change looks bigger than it is. I wrapped a `React.memo` around the export so that the component wouldn't rerender when `searchParams` of the url changes.